astropy/io/misc/tests/test_hdf5.py

```diff
@@ -395,7 +401,7 @@ def test_preserve_serialized(tmpdir):
     assert t1.meta == t2.meta
 
 
-@pytest.mark.skipif('not HAS_H5PY')
+@pytest.mark.skipif('not HAS_H5PY' or 'not HAS_YAML')
```

Comment:
Using a string seems not necessary: https://pytest.org/latest/skipping.html#specifying-conditions-as-strings-versus-booleans (astropy bundles version 2.5 and using a boolean condition is possible since pytest 2.4)